jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -2324,6 +2301,28 @@
 						"description": ""
 					},
 					"response": []
+				},
+				{
+					"name": "deploy conf with resource template name",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 200\"] = responseCode.code === 200;"
+								]
+							}
+						}
+					],
+					"request": {
+						"url": "{{url}}/v1.0/applications/{{HealthCheckApp}}/conf/hct.xml;groupName={{HealthCheckGroup}};jvmName=N9SF-LTST-{{HealthCheckGroup}}-{{host}}-1",
```

Comment:
It looks like this test is still dependent on a health check JVM. Can we run the same test on a postman specific JVM since we're making changes?